### PR TITLE
README: add instructions for python client installation on Fedora

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution, We already documented this in the Python client repository's README, I don't think it is necessary here again (as the global non-platform specific `pip` command is already documented).

---

**Edit.** Since we already document `brew` for Linux and Mac users, maybe for Windows users, we could mention tlrc's Winget manifest i.e. `winget install tldr`(for those who don't want to additionally install NodeJS just to use tldr).

**Comment by fazlearefin** [comment]
Here is a screendump:

```shell
[root@56a94576f237 ~]# lsb_release -a
LSB Version:    n/a
Distributor ID: Fedora
Description:    Fedora Linux 39 (Container Image)
Release:        39
Codename:       n/a
[root@56a94576f237 ~]# dnf search tldr
Last metadata expiration check: 0:00:45 ago on Fri Feb 16 17:32:01 2024.
========================================================================================================= Name Exactly Matched: tldr ==========================================================================================================
tldr.noarch : Simplified and community-driven man pages
============================================================================================================= Name Matched: tldr ==============================================================================================================
python3-pytest-tldr.noarch : Pytest plugin that limits the output to just the things you need
============================================================================================================ Summary Matched: tldr ============================================================================================================
tealdeer.x86_64 : Fetch and show tldr help pages for many CLI commands
[root@56a94576f237 ~]# dnf info tldr
Last metadata expiration check: 0:00:56 ago on Fri Feb 16 17:32:01 2024.
Available Packages
Name         : tldr
Version      : 3.2.0
Release      : 3.fc39
Architecture : noarch
Size         : 35 k
Source       : tldr-3.2.0-3.fc39.src.rpm
Repository   : fedora
Summary      : Simplified and community-driven man pages
URL          : https://github.com/tldr-pages/tldr-python-client
License      : MIT
Description  : A Python command line client for tldr - Simplified and community-driven
             : man pages http://tldr-pages.github.io/.

[root@56a94576f237 ~]# dnf install tldr -y
Last metadata expiration check: 0:01:07 ago on Fri Feb 16 17:32:01 2024.
Dependencies resolved.
===============================================================================================================================================================================================================================================
 Package                                                         Architecture                                         Version                                                       Repository                                            Size
===============================================================================================================================================================================================================================================
Installing:
 tldr                                                            noarch                                               3.2.0-3.fc39                                                  fedora                                                35 k
Installing dependencies:
 python3-colorama                                                noarch                                               0.4.6-4.fc39                                                  fedora                                                71 k
 python3-shtab                                                   noarch                                               1.6.1-3.fc39                                                  fedora                                                43 k
 python3-termcolor                                               noarch                                               2.3.0-3.fc39                                                  fedora                                                20 k

Transaction Summary
===============================================================================================================================================================================================================================================
Install  4 Packages

Total download size: 169 k
Installed size: 414 k
Downloading Packages:
(1/4): python3-termcolor-2.3.0-3.fc39.noarch.rpm                                                                                                                                                               160 kB/s |  20 kB     00:00
(2/4): python3-shtab-1.6.1-3.fc39.noarch.rpm                                                                                                                                                                   325 kB/s |  43 kB     00:00
(3/4): python3-colorama-0.4.6-4.fc39.noarch.rpm                                                                                                                                                                509 kB/s |  71 kB     00:00
(4/4): tldr-3.2.0-3.fc39.noarch.rpm                                                                                                                                                                            945 kB/s |  35 kB     00:00
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                                                                                                                          158 kB/s | 169 kB     00:01
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                                                                                                                       1/1
  Installing       : python3-termcolor-2.3.0-3.fc39.noarch                                                                                                                                                                                 1/4
  Installing       : python3-shtab-1.6.1-3.fc39.noarch                                                                                                                                                                                     2/4
  Installing       : python3-colorama-0.4.6-4.fc39.noarch                                                                                                                                                                                  3/4
  Installing       : tldr-3.2.0-3.fc39.noarch                                                                                                                                                                                              4/4
  Running scriptlet: tldr-3.2.0-3.fc39.noarch                                                                                                                                                                                              4/4
  Verifying        : python3-colorama-0.4.6-4.fc39.noarch                                                                                                                                                                                  1/4
  Verifying        : python3-shtab-1.6.1-3.fc39.noarch                                                                                                                                                                                     2/4
  Verifying        : python3-termcolor-2.3.0-3.fc39.noarch                                                                                                                                                                                 3/4
  Verifying        : tldr-3.2.0-3.fc39.noarch                                                                                                                                                                                              4/4

Installed:
  python3-colorama-0.4.6-4.fc39.noarch                           python3-shtab-1.6.1-3.fc39.noarch                           python3-termcolor-2.3.0-3.fc39.noarch                           tldr-3.2.0-3.fc39.noarch

Complete!
[root@56a94576f237 ~]# tldr --version
tldr 3.2.0 (Client Specification 1.5)
[root@56a94576f237 ~]#

```

**Comment by acuteenvy** [comment]
> mention tlrc's Winget manifest that I [package](https://github.com/microsoft/winget-pkgs/tree/master/manifests/t/tldr-pages/tlrc) i.e. `winget install tldr` [Note: this is an alias/moniker to `winget install tldr-pages.tlrc`]

If you decide to add this, I'd strongly recommend using `tldr-pages.tlrc` instead. The `tldr` package just confuses everyone - each repository packages a different client under that name. In my opinion, using a unique name for each client would be a lot better.

**Comment by vitorhcl** [comment]
Thank you for your contribution and for rasing this topic! IMO #12287 is a better solution for avoiding duplicate documentation and updating all places when a change occurs.

**Comment by fazlearefin** [comment]
Closing this PR as the discussion has moved to https://github.com/tldr-pages/tldr/pull/12287
